project/1.18/assets/tinkers-construct/tconstruct/book/puny_smelting/zh_cn/further_reading/encyclopedia.json

```
@@ -7,7 +7,7 @@
   },
   "text": [
     {
-      "text": "许多工匠空有满腔抱负，却因为胳膊太少，做不到一边拿着两本书一边造工具。但如果你像我一样长四只胳膊，问题就会迎刃而解！幸运的是，对于你们这些书呆子来说，只要少量熔融黄金，你就可以制作Selena所著的新书，里面包含了有关制作工具的一切内容。但是不要在那本书里翻找关于我牛逼的冶炼炉的内容，你找不到的。"
+      "text": "显然，一些有志向的工匠没有足够多的手臂来在制作工具的同时拿着好几本书。但如果你像我一样只有四只胳膊，那将不是问题！幸运的是，对于你们这些书呆子来说，只要一点点熔融黄金，你就可以制作Selena所著的新书，里面包含你需要的所有有关于制作工具的相关信息。但是不要指望在那里找到任何我所讲解的关于冶炼炉的部分。"
```

```suggestion
      "text": "许多工匠空有满腔抱负，却因为胳膊太少，做不到一边拿着两本书一边造工具。但如果你像我一样长四只胳膊，问题就会迎刃而解！幸运的是，对于你们这些书呆子来说，只要少量熔融黄金，你就可以制作Selena所著的新书，里面包含了有关制作工具的一切内容。但是不要在那本书里翻找关于我牛逼的冶炼炉的内容，你找不到的。"
```
除了“牛逼”可以换个词，剩下的可以说是大幅改进